app/models/export.rb

```diff
@@ -20,6 +20,16 @@ def to_following_accounts_csv
     to_csv account.following
   end
 
+  def to_favourite_posts_csv
+    CSV.generate do |csv|
+      account.favourites.each do |favourite|
+        domain = favourite.status.account.local? ? Rails.configuration.x.local_domain : favourite.status.account.domain
+
+        csv << [domain, favourite.status.account.username, favourite.status_id]
```

Comment:
This will export a local status ID, which is irrelevant to any other instance but the one where it comes from. It would make sense to just export a list of URIs here. Though mind you, it's impossible to do a look up by URI if you don't already have the status in the DB. So list of URLs would be the only thing that worked.